Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:2
```

Comment:
Mind using `python:2-slim` here instead, as the resulting docker image size difference is pretty huge

```
$ docker images | grep elasticstat
objectrocket/elasticstat                                python2-slim                               5a2e1166a522        5 seconds ago       156MB
objectrocket/elasticstat                                python2                                    9170d29364f8        2 minutes ago       899MB
```

Reply:
Done